sharktank/sharktank/utils/export_artifacts.py

```diff
@@ -407,16 +407,17 @@ def compile_to_vmfb(
             compile_args += [
                 f"--iree-hal-dump-executable-files-to={hal_dump_path}/files"
             ]
+
+        compile_args += [
```

Comment:
This will force these values of the flags. I don't think it will be possible to export **without**, e.g., `--iree-opt-level=O3`.

Reply:
You can specify in your extra compile args
```
--iree-opt-level=O2
```

and it would take that as the optimization level.

Reply:
It will take the last value rather than error out about duplicate flags?